### PR TITLE
Plugins.md - Leaflet.Control.Select added to the list of plugins

### PR DESCRIPTION
**Pull request by adammertel**
https://github.com/adammertel/Leaflet.Control.Select

### CODE REVIEW

**Comment by johnd0e** [comment]
@adammertel 
Please resolve conflicts

File: docs/plugins.md [hunk 1]
```diff
@@ -4190,8 +4190,8 @@ Buttons, sliders, toolbars, sidebars, and panels.
 			<a href="https://github.com/publiclab">Public Lab</a>
 		</td>
 	</tr>
-	<tr>
-		<td>
+  <tr>
+    <td>
```
Comment (johnd0e) [inline docs/plugins.md:4194]: Please restore tabs

File: docs/plugins.md [hunk 2]
```diff
@@ -4202,6 +4202,18 @@ Buttons, sliders, toolbars, sidebars, and panels.
 			<a href="https://github.com/ptma">JJ Jin</a>
 		</td>
 	</tr>
+  <tr>
```
Comment (johnd0e) [inline docs/plugins.md:4205]: tabs

File: docs/plugins.md [hunk 3]
```diff
@@ -4202,6 +4202,18 @@ Buttons, sliders, toolbars, sidebars, and panels.
...
+		<td>
+			<a href="https://github.com/adammertel/Leaflet.Control.Select">Leaflet.Control.Select
+			</a>
+		</td>
+		<td>
+			 Customisable menu-style control, allows menu-items grouping or multiple-selection mode. See <a href="https://adammertel.github.io/Leaflet.Control.Select/">demo</a>.
```
Comment (johnd0e) [inline docs/plugins.md:4211]: Excessive space

File: docs/plugins.md [hunk 4]
```diff
@@ -4202,6 +4202,18 @@ Buttons, sliders, toolbars, sidebars, and panels.
...
+		</td>
+		<td>
+			<a href="https://github.com/adammertel">Adam Mertel</a>
+    </td>
+  </tr>
```
Comment (johnd0e) [inline docs/plugins.md:4216]: tabs

**Review by johnd0e — CHANGES_REQUESTED** [review]

**Review by jonkoops — COMMENTED** [review]
Can we get rid of the unrelated white-space changes?

File: docs/plugins.md [hunk 5]
```diff
@@ -4190,8 +4190,8 @@ Buttons, sliders, toolbars, sidebars, and panels.
 			<a href="https://github.com/publiclab">Public Lab</a>
 		</td>
 	</tr>
-	<tr>
-		<td>
+  	<tr>
+    	<td>
```
Comment (johnd0e) [inline docs/plugins.md:4194]: Tabs

File: docs/plugins.md [hunk 6]
```diff
@@ -4202,6 +4202,18 @@ Buttons, sliders, toolbars, sidebars, and panels.
 			<a href="https://github.com/ptma">JJ Jin</a>
 		</td>
 	</tr>
+  	<tr>
```
Comment (johnd0e) [inline docs/plugins.md:4205]: Tabs

File: docs/plugins.md [hunk 7]
```diff
@@ -4202,6 +4202,18 @@ Buttons, sliders, toolbars, sidebars, and panels.
...
+			<a href="https://github.com/adammertel/Leaflet.Control.Select">Leaflet.Control.Select
+			</a>
+		</td>
+		<td>
+			Customisable menu-style control. See <a href="https://adammertel.github.io/Leaflet.Control.Select/">demo</a>.
+		</td>
+		<td>
+			<a href="https://github.com/adammertel">Adam Mertel</a>
+		</td>
+  	</tr>
```
Comment (johnd0e) [inline docs/plugins.md:4216]: Tabs

**Review by johnd0e — CHANGES_REQUESTED** [review]

**Comment by adammertel** [comment]
Sorry guys, it takes so long. Hope it is good now.

**Review by mourner — APPROVED** [review]

**Pull request merged**